### PR TITLE
Fix AWS REGION configuration

### PR DESCRIPTION
**Pull request by simon0191**
The S3 store and the Download URL were created without taking into account AWS REGION env var.

### CODE REVIEW

**Review by mattyr — CHANGES_REQUESTED** [review]
great stuff, thanks!  i think that we can probably make "region" a field on the s3store struct, and add it as a parameter in NewS3Store, and inside that function fall back to the environment variable and finally the default.

**Comment by simon0191** [comment]
👍 working on it

**Comment by simon0191** [comment]
@mattyr Done

File: store.go [hunk 1]
```diff
@@ -63,3 +66,13 @@ func (s *s3store) Delete(appId string, backupId string) error {
 func (*s3store) pathFor(appId string, backupId string) string {
 	return fmt.Sprintf("pgbackups/%s/%s.backup", appId, backupId)
 }
+
+func setRegion(region string) string {
```
Comment (mattyr) [inline store.go]: let's rename this to "getRegion" for clarity, and arg name should be "regionName" or maybe just "name" for consistency

**Review by mattyr — CHANGES_REQUESTED** [review]
perfect, maybe just a minor name change and this will be gtg.  thanks so much!

**Comment by mattyr** [comment]
thanks @simon0191

**Pull request merged**